slack-base/src/main/java/com/hubspot/slack/client/models/users/SlackUserIF.java

```diff
@@ -39,6 +39,9 @@
   @JsonProperty("updated")
   Optional<Integer> getRawUpdated();
 
+  @JsonProperty("locale")
+  Optional<String> getLocale();
```

Comment:
`@JsonProperty` is useful if the method name doesn't match the property name. Since we're using `getLocale` here, it should be serialized into `locale`. So we should be fine without the `@JsonProperty` annotation

Reply:
Awesome, I'll take that out then and test